etc/kayobe/kolla/globals.yml

```diff
@@ -26,40 +26,44 @@ monasca_agent_authorized_roles:
 # Default size causes ES to fall over very easily
 es_heap_size: "24g"
 
+# Settings to ensure backwords compatibility with Queens Monasca deployment
+monasca_grafana_server_port: "3000"
```

Comment:
Need to be careful not to apply this config since the monasca-grafana image currently contains a bug which messes up the graph legends for users of the system. Currently, the old instance of Grafana is running until this bug is fixed.

Reply:
I'm not planning to run any skip tags for the rocky upgrade - we need the config to reflect the system's state. Is there a change we need to make to the config for that to work?

Reply:
No problem, I just added commit for those changes.

Reply:
Thanks